server/controllers/ArticleController.js

```diff
@@ -0,0 +1,279 @@
+import slug from 'slug';
+import uuid from 'uuid';
+
+import { Article } from '../db/models';
+
+const userId = 1;
+
+/**
+ *
+ *
+ * @class ArticleController
+ *
+ * @desc This is a class controller
+ * that handles all operations related to Articles.
+ */
+class ArticleController {
+  /**
+   *@description Add a new Article to the Article list
+   *
```

Comment:
#### Nitpick
Remove this blank line. Also remove other  blank lines in comment blocks